pontoon/contributors/static/js/profile.js

```diff
@@ -296,8 +296,12 @@ var Pontoon = (function (my) {
             .find('.selector .value')
             .html($(this).html());
 
+          // Rerender the show more button in case it was hidden
+          $('#show-more').show();
+
           const type = $('#contributions .type-selector span').data('type');
           const user = $('#server').data('user');
+          let months_shown = $(this).data('months_shown');
```

Comment:
Nit: Note that we use camelCase in JS, kebab-case in HTML and snake_case in python. 🥳